dial.go

```diff
@@ -19,6 +19,11 @@ import (
 	msmux "github.com/multiformats/go-multistream"
 )
 
+// DialTimeout is the maximum duration a Dial is allowed to take.
+// This includes the time between dialing the raw network connection,
+// protocol selection as well the handshake, if applicable.
+var DialTimeout = 60 * time.Second
```

Comment:
60 seconds now? Didnt this use to be 30 seconds?

Reply:
10 seconds, I think. However, this *also* includes secio negotiation (and now matches the accept timeout).

Honestly, they should probably *both* be 10.

Reply:
Changed back to 60. This is just a fallback timeout and should be conservative (users can always pass a context with a lower timeout).